playwright/plugins/docker/index.ts

```diff
@@ -74,6 +75,16 @@ export class Docker {
             }
         }
 
+        if (isPodman) {
+            // Make host.containers.internal work to allow the container to talk to other services via host ports.
+            params.push("--network");
+            params.push("slirp4netns:allow_host_loopback=true");
+        } else {
+            // Make host.docker.internal work to allow the container to talk to other services via host ports.
```

Comment:
this comment is still wrong.

Suggest pulling the `host.containers.internal` version out of the `if` block.